aictl/cli/main.py

```diff
@@ -403,6 +449,46 @@ def main():
     )
     t2a_parser.set_defaults(func=t2a)
 
+    ####Upscale
+    upscale_parser = subparsers.add_parser("upscale", help="Upscale an image")
+    upscale_parser.add_argument("-p", "--prompt", default="", help="the prompt to use")
+    upscale_parser.add_argument(
+        "-i", "--image", default=None, help="the local image file to edit"
+    )
+    upscale_parser.add_argument(
+        "-u",
+        "--image-url",
+        default="https://raw.githubusercontent.com/timothybrooks/instruct-pix2pix/main/imgs/example.jpg",
+        help="the url of the image to edit",
+    )
+    upscale_parser.add_argument(
+        "-y",
+        "--scheduler",
+        default="euler",
+        help="available schedulers are: lms, ddim, dpm, euler, pndm, ddpm, and eulera",
+        type=scheduler_validator,
+    )
+    upscale_parser.add_argument(
+        "-o",
+        "--output-path",
+        default=f"output_upscale{utc_time}.png",
+        help="the path for audio when generation is complete",
+    )
+    upscale_parser.add_argument(
+        "-m",
+        "--model",
+        default="esrgan",
+        help="the upscale model (x4) to use (options: esrgan,sdx4:use this on your own risk.OOM danger!!)",
```

Comment:
You can remove the `:use this on your own risk.OOM danger!!` part. This whole repo has danger of OOM haha

Reply:
done